pages/homePage.js

```diff
@@ -2,18 +2,11 @@ const { I } = inject();
 
 module.exports = {
 
-    websiteLanguageSpoiler: { xpath: '//*[@id="form-language"]/div/span' },
-    websiteLanguageEng: { xpath: '//*[@id="form-language"]/div/ul/li[1]/button' },
     myAccountSpoiler: { xpath: '//*[@id="top-links"]/ul/li/span/span' },
     registerLink: { xpath: '//*[@id="top-links"]/ul/li/ul/li[1]/a' },
```

Comment:
this locator is ugly
![image](https://user-images.githubusercontent.com/19595492/219250723-4698e2ae-64db-41c3-b6e3-0c50c605d3bb.png)